sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/RecoveryServicesVaultResource.cs

```diff
@@ -443,20 +445,176 @@ public virtual ArmOperation<RecoveryServicesVaultResource> Update(WaitUntil wait
             }
         }
 
+        /// <summary>
+        /// Gets the operation status for a resource.
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Request Path</term>
+        /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/operationStatus/{operationId}</description>
+        /// </item>
+        /// <item>
+        /// <term>Operation Id</term>
+        /// <description>Vault_GetOperationStatus</description>
+        /// </item>
+        /// <item>
+        /// <term>Default Api Version</term>
+        /// <description>2025-02-01</description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        /// <param name="operationId"> The <see cref="string"/> to use. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentException"> <paramref name="operationId"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="operationId"/> is null. </exception>
+        public virtual async Task<Response<OperationResource>> GetOperationStatusRecoveryServicesClientAsync(string operationId, CancellationToken cancellationToken = default)
```

Comment:
this looks like a LRO polling operation - we should remove this operation.
Please add a `@@scope` decorator to hide this.